src/core/on-chain/index.ts

```diff
@@ -145,217 +145,212 @@ export const OnChainMixin = (superclass) =>
         /// TODO: unable to check balance.total_in_BTC vs this.dustThreshold at this point...
       }
 
-      return await redlock(
-        { path: this.user._id, logger: onchainLogger },
-        async (lock) => {
-          const balance = await this.getBalances(lock)
-          onchainLogger = onchainLogger.child({ balance })
-
-          // quit early if balance is not enough
-          if (balance.total_in_BTC < amount) {
-            throw new InsufficientBalanceError(undefined, { logger: onchainLogger })
-          }
+      return redlock({ path: this.user._id, logger: onchainLogger }, async (lock) => {
```

Comment:
not sure why github is not showing correctly the diff because of this ... did you change any logic?

Reply:
No logic changes here. Only removed the return await but I think Prettier is redoing the indentation here.